src/index.js

```diff
@@ -32,6 +32,7 @@ const applyConfig = (config) => {
   config.settings.allowed_cors_destinations = [
     ...(config.settings.allowed_cors_destinations || []),
     'www.eea.europa.eu',
+    'http://alin.dev2aws.eea.europa.eu',
```

Comment:
This should not be here. I think `eea.europa.eu` should work for all cases.

Reply:
Done!